src/change_log.js

```diff
@@ -154,12 +178,28 @@ function prepareChangeLog(gitMessages, scopes) {
     //     }
     // });
 
-    const releaseChangelog = changeLogForSlack(changelogStructure, scopes);
+    const { releaseChangelog, releaseChangelogV2 } = await changeLogForSlack(changelogStructure, scopes);
 
     core.info('Change log was generated successfully');
-    return releaseChangelog;
+    return { releaseChangelog, releaseChangelogV2 };
+}
+
+async function improveChangeLog(changeList) {
+    if (!process.env.OPEN_AI_TOKEN) throw new Error('Cannot improve changelog, missing OPEN_AI_TOKEN env variable.');
+    const completion = await openai.createCompletion({
+        model: 'text-davinci-003',
+        prompt: `${OPEN_AI_IMPROVE_CHANGELOG_REQUEST}\n${changeList.map((line) => `* \`${line}\``).join('\n')}`,
+        max_tokens: 512,
+        temperature: 0.5,
+    });
```

Comment:
I would set some reasonable timeout here, just in case, so that the changelog creation does not take too long in case of network issues: https://github.com/openai/openai-node#request-options
```suggestion
    }, {
        timeout: 10000,
    });
```